cmd/kpod/rm.go

```diff
@@ -39,20 +44,43 @@ func rmCmd(c *cli.Context) error {
 	defer runtime.Shutdown(false)
 
 	args := c.Args()
-	if len(args) == 0 {
+	if len(args) == 0 && !c.Bool("all") {
 		return errors.Errorf("specify one or more containers to remove")
 	}
 
-	for _, container := range args {
+	var delContainers []string
+	if c.Bool("all") {
+		allContainers, err := runtime.GetContainers()
+		if err != nil {
+			return errors.Wrapf(err, "unable to get container list")
+		}
+		for _, container := range allContainers {
+			delContainers = append(delContainers, container.ID())
+		}
+	} else {
+		delContainers = args
+	}
+	var lastError error
+	for _, container := range delContainers {
 		ctr, err := runtime.LookupContainer(container)
 		if err != nil {
-			return errors.Wrapf(err, "error looking up container", container)
+			if err != nil {
```

Comment:
I think you have an extra `if err != nil` here

Reply:
fixed